scripts/release/install_release_script_dependencies.R

```diff
@@ -25,7 +25,14 @@ if(!"cli" %in% installed.packages()) {
 library(cli)
 cli_alert_info("Installing packages")
 
-packages <- c("getPass", "arrow", "jsonlite", "future", "MolgenisArmadillo", "DSI", "devtools", "resourcer", "DSMolgenisArmadillo", "RCurl")
+packages <- c("diffobj", "getPass", "arrow", "jsonlite", "future",
+  "RCurl",
+  "devtools",
+  "DSI",
+  "resourcer",
+  "MolgenisArmadillo",
+  "DSMolgenisArmadillo"
+)
```

Comment:
Ordered in least changing to most changing order. We could split this in staging scripts for more image layers ... not sure that is very useful but on MacOS it took ages to build the image